src/components/Input/Input.tsx

```diff
@@ -0,0 +1,39 @@
+import cn from "classnames";
+
+import "./input.scss";
+
+type Props = {
+  placeholder: string;
+  name: string;
+  value: any;
+  onChange: any;

```

Comment:
You can see what you're passing to this function where it was called in [here](https://github.com/MarcelloAragoni/todo-list/pull/2/files#diff-0414a1d786fc54ea337f45079b8c5d47254051dfc247a8f39d0f4c474fcb9f12R27)


1. Take a look where the function is called (Line 27)
2. Put the mouse over the `value` at line 27, and you'll see what is its value (string)
3. That will be your `type` for the `onChange` argument:

```suggestion
  onChange: (value: string) => void;
```